packages/airnode-node/src/providers/actions.ts

```diff
@@ -71,23 +80,23 @@ async function processEvmProviderRequests(
 }
 
 export async function processRequests(
-  providerStates: ProviderStates,
+  sponsorProviderStates: ProviderState<EVMProviderSponsorState>[],
```

Comment:
I dislike that this function is coupled with the evm providers (ideally it should be chain agnostic), but since Burak mentioned we might end up using second party oracles for non EVM chains this might not matter that much.